python/tidy/servo_tidy/tidy.py

```diff
@@ -788,6 +821,17 @@ def get_file_list(directory, only_changed_files=False, exclude_dirs=[]):
 
 
 def scan(only_changed_files=False, progress=True):
+    # check if config file exists
+    if not os.path.exists(CONFIG_FILE_PATH):
+        print("%s config file is required but was not found" % CONFIG_FILE_PATH)
+        sys.exit(1)
+    # load configs from ./servo-tidy.toml
+    with open(CONFIG_FILE_PATH) as content:
+        conf_file = content.read()
+    # check config file for errors
+    config_errors = check_config_file(conf_file, progress)
+    # parse config file
+    parse_config(conf_file)
```

Comment:
This is probably a bad idea. We should revert this back to `config_errors = check_config_file(progress)`, and move `parse_config` back into `check_config_file`. But now, we don't have to check whether it's our actual `CONFIG_FILE_PATH`, because the source of the config errors is in `check_config_file` itself, and not in `parse_config`. This means we'll get errors before we reach the line where we parse the config. So, voila! No test-specific code in tidy :)